synapse/cortex.py

```diff
@@ -504,7 +515,7 @@ def setFeedOffs(self, iden, offs):
         Return stream of (iden, provenance stack) tuples at the given offset.
         '''
         count = 0
-        async for iden, stack in self.cell.layer.provStacks(offs, size):
+        for iden, stack in self.cell.provstor.provStacks(offs, size):
```

Comment:
A CoreApi should nearly always expose APIs from the cortex as directly as possible.  For the prov APIs implemented in CoreApi, create a Cortex API layer that hides the deref'd provstor guts ( using the API locally should not require making a proxy and should also not require special knowledge that is any different than calling the API remotely ).  Probably just move this API down into cortex and wrap it here.

Reply:
discussed and deferring.